Mathlib/LinearAlgebra/Dimension.lean

```diff
@@ -108,6 +108,9 @@ protected irreducible_def Module.rank : Cardinal :=
   ⨆ ι : { s : Set V // LinearIndependent K ((↑) : s → V) }, (#ι.1)
 #align module.rank Module.rank
 
+instance : Nonempty { s : Set V // LinearIndependent K ((↑) : s → V) } :=
```

Comment:
Can you add a comment explaining why this instance is useful? I don't think it is particularly reasonably to add such an instance for every predicate that could ever exist!